presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -335,35 +388,26 @@ public void dropTable(ConnectorTableHandle tableHandle)
         }
     }
 
-    @Override
-    public HiveOutputTableHandle beginCreateTable(ConnectorSession session, ConnectorTableMetadata tableMetadata)
+    private void getColumnInfo(ConnectorTableMetadata tableMetadata,
```

Comment:
This refactoring makes the diff harder to read. Can you split it into a separate commit?